cli/command/container/stats_helpers.go

```diff
@@ -46,15 +46,15 @@ func (s *stats) remove(id string) {
 
 func (s *stats) isKnownContainer(cid string) (int, bool) {
 	for i, c := range s.cs {
-		if c.Name == cid {
+		if c.Container == cid {
 			return i, true
 		}
 	}
 	return -1, false
 }
 
 func collect(s *formatter.ContainerStats, ctx context.Context, cli client.APIClient, streamStats bool, waitFirst *sync.WaitGroup) {
-	logrus.Debugf("collecting stats for %s", s.Name)
+	logrus.Debugf("collecting stats for %s", s.Container)
```

Comment:
@vdemeester yes, now I see it. s.Container can be the ID or the Name depends on what the user enters.